index.Rmd

```diff
@@ -1,59 +1,57 @@
 ---
+pagetitle: "WUR Geoscripting course"
 author: "Jan Verbesselt and Dainius Masiliunas"
 date: "`r format(Sys.time(), '%d %B, %Y')`"
-pagetitle: "WUR Geoscripting course"
 output:
   knitrBootstrap::bootstrap_document:
+    title: "WUR Geoscripting course"
     theme: "simplex"
     highlight: Tomorrow Night Bright
     menu: FALSE
     theme.chooser: TRUE
     highlight.chooser: FALSE
 ---
-<style type="text/css">body {max-width: none;}
-a:visited {
-      color: #91170a;
-    }
+
+<style type="text/css">
+body {max-width: none;}
+a:visited {color: #91170a;}
 </style>
 
 # [WUR Geoscripting](https://geoscripting-wur.github.io/) <img src="https://www.wur.nl/upload/854757ab-168f-46d7-b415-f8b501eebaa5_WUR_RGB_standard_2021-site.svg" alt="WUR logo" style="height: 35px;"/>
 
-
 We recommend refreshing your R and Python knowledge before the course with
 
 #### [DataCamp: Interactive learning ](https://www.datacamp.com/) ![DataCamp](figs/datacamp.svg)
 
 
 # Week 1
 
-
-#### [Lesson 1: Carrying out a scripting project](http://geoscripting-wur.github.io/RProjectManagement)
-#### [Lesson 2: Introduction to Linux](http://geoscripting-wur.github.io/Intro2Linux)
-#### [Lesson 3: Intro to functions and refresher on R](https://geoscripting-wur.github.io/Scripting4GeoIntro/)
-#### Assignment of Week 1
-#### [Lesson 4: Intro to raster](http://geoscripting-wur.github.io/IntroToRaster)
-<!--#### [Lesson 2: Overview of R for geoscripting](https://geoscripting-wur.github.io/Scripting4Geo/)-->
+#### [Tutorial 1: Linux & version control](https://geoscripting-wur.github.io/RProjectManagement)
+#### [Tutorial 2: Linux terminal & Bash](https://geoscripting-wur.github.io/Intro2Linux)
+#### [Tutorial 3: Intro to functions and refresher on R](https://geoscripting-wur.github.io/Scripting4GeoIntro/)
+#### [Tutorial 4: Intro to raster](https://geoscripting-wur.github.io/IntroToRaster)
```

Comment:
Would it work to make the links relative, i.e. just `RProjectManagement.html`?

Reply:
It would make it more portable if we ever want to switch from GitHub pages to something else (think of a MOOC platform). But not the highest priority right now.